brave/brave-core
# Transition Brave Ads notification ads from notifications.json to base::Value::List

Resolves https://github.com/brave/brave-browser/issues/24598

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm clicking a notification ad in the system tray on Android opens a tab with the ad
- Confirm clicking a notification ad opens a tab with the ad
- Confirm users are rewarded for notification ads
